storage/local/mapper_test.go

```diff
@@ -62,7 +62,7 @@ func TestFPMapper(t *testing.T) {
 	p, closer := newTestPersistence(t, 1)
 	defer closer.Close()
 
-	mapper, err := newFPMapper(sm, p)
+	mapper, _ := newFPMapper(sm, p)
```

Comment:
We should not ignore the error but t.Fatal on it.

Reply:
Sure, I can fix those.